Sources/ProjectDescription/TestAction.swift

```diff
@@ -44,7 +71,7 @@ public struct TestAction: Equatable, Codable {
     ///   - diagnosticsOptions: Diagnostics options.
     ///   - language: Language (e.g. "pl")
     ///   - region: Region (e.g. "PL")
-    public init(targets: [TestableTarget] = [],
+    public init(targets: [TestableTarget],
```

Comment:
ah sorry just spotted this, its minor, I will re-add the default empty list to maintain backward source compatibility `= []`

Reply:
ah ok. I was confused about this since it didn‘t seem to make any sense to have no targets.